docs/_library/jdm-classics.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: experiment
+name:  "jdm-classics"
+maintainer: "@vsoch"
```

Comment:
Do you want to put your GitHub alias here?

Reply:
Yes, that works for me.

Reply:
Great! Could you please update the PR for this experiment file and the previously merged one?